### PR TITLE
Changes Benefits headers to use Write

### PR DESCRIPTION
**Pull request by logangingerich**
Updates three instances of `Benefits` to use `Write` instead, for clarity.

### CODE REVIEW

**Comment by alexromano** [comment]
Would prefer if we did not make this change right now. Specifying Benefits explicitly aligns with the rest of our docs (we don't use "write" language anywhere else) and with the current evolution of the product (Benefits, and everything else). Its not perfect language, but we should hold off on changing it until we restructure our communication around assisted/automated

**Comment by tylernix** [comment]
@alexromano I had @logangingerich make this change for me. Every day, I get people confused by why "Benefits" isn't supported for a provider that clearly supports benefits in their system. Considering Benefits is our **only** write functionality at the moment, it makes more sense to explain that it is Read vs Write. I would propose we either make it "Read vs Write" or "Read Census + Pay Data vs Write Benefits".

I have been wanting to make this change for months. The current wording is confusing A LOT of customers reading our documentation.

File: docs/Development-Guides/Providers.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ title: HRIS
 > **Read:** The `Company`, `Directory`, `Individual`, `Employment`, `Payment` and `Pay Statement` endpoint are read-only endpoints. Providers can be either an [Automated API Provider](../Product-Guides/Automated-Connect-Flow.md) or an [Assisted API Provider](../Product-Guides/Assisted-Connect-Flow.md).
 
 <!-- theme: info -->
-> **Benefits:** The benefits endpoints contain both read and write functionalities. Providers can be either an Automated API Benefit Provider or an Assisted API Benefit Provider. Only `Create Benefit`, `Update Benefit`, `Enroll Individuals in Benefits` and `Unenroll Individuals from Benefits` endpoints are available for Assisted API Benefit Providers.
```
Comment (tylernix) [inline docs/Development-Guides/Providers.md:16]: This needs to be "Benefits (Write)"

**Comment by tylernix** [comment]
@logangingerich Please also change the `**Last Updated:** 2023-01-20` to today's date.

File: docs/Development-Guides/Providers.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ title: HRIS
 > **Read:** The `Company`, `Directory`, `Individual`, `Employment`, `Payment` and `Pay Statement` endpoint are read-only endpoints. Providers can be either an [Automated API Provider](../Product-Guides/Automated-Connect-Flow.md) or an [Assisted API Provider](../Product-Guides/Assisted-Connect-Flow.md).
```
Comment (tylernix) [inline docs/Development-Guides/Providers.md:13]: Please also change `**Last Updated:** 2023-01-20` in line 3 to today's date

**Review by tylernix — CHANGES_REQUESTED** [review]

File: docs/Development-Guides/Providers.md (hunk 1 above)
Comment (logangingerich) [reply docs/Development-Guides/Providers.md:16]: Just pushed the change

File: docs/Development-Guides/Providers.md (hunk 2 above)
Comment (logangingerich) [reply docs/Development-Guides/Providers.md:13]: 🤦  Totally forgot that, just added in the update.

**Review by tylernix — APPROVED** [review]

**Comment by alexromano** [comment]
@tylernix i would prefer the latter then: "Read Census + Pay Data vs Write Benefits"

**Review by alexromano — APPROVED** [review]

**Pull request merged**